docs/servicenow.md

```diff
@@ -0,0 +1,149 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+with a ServiceNow object. A list of the ServiceNow fields that have been
+tested when creating and editing tickets will be included. Because each
+instance of ServiceNow can have custom fields and custom values, some
+of the tested fields may not be applicable to certain instances of ServiceNow.
+Additionally, your ServiceNow instance may contain ticket fields that we have
+not tested. Keep in your mind different ServiceNow tables contain different
+fields with different properties. Mandatory field of one table is optional
+in another table (or it doesn't exist at all).
+
+Custom field names and values can be passed in as keyword arguments when
+creating and editing tickets. These are processed by the ServiceNow REST API
+right afterwards. You can also use [get_ticket_content](#content) to get ticket
+content including preset field names.
+
+For more information about REST API and fields see ServiceNow's Wiki:
+http://wiki.servicenow.com/index.php?title=REST_API
+http://wiki.servicenow.com/index.php?title=Introduction_to_Fields
+http://wiki.servicenow.com/index.php?title=Dot-Walking
+
+
+## Table of contents
+- [create()](#create)
+- [get_ticket_content()](#content)
+- [edit()](#edit)
+- [add_comment()](#comment)
+- [change_status()](#status)
+- [add_cc()](#add_cc)
+- [rewrite_cc()](#rewrite_cc)
+- [remove_cc()](#remove_cc)
+
+### create(self, short_description, description, category, item, \*\*kwargs) <a name="create"></a>
+
+Creates a ticket. The required parameters for ticket creation are
+short description and description of the issue. Keyword arguments are used
+for other ticket fields.
+
+```python
+t.create(short_description='Ticket summary',
+         description='Ticket description')
+```
+
+Tested create() ticket fields:
```

Comment:
I think noting the fields we've tested is important. The docs/ folder gives a quick overview of all the supported methods along with the supported fields for those methods. The examples/ folder contains more detailed code examples for example workflows using the tool (ex. create a ticket, add a comment, change to closed, close requests session).